### PR TITLE
Commands v2

### PR DESCRIPTION
**Pull request by BindyaBhaskar**
Added CLI support for Commands V2.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1931,6 +1931,181 @@ parent       <uuid>
...
+#### Sub commands
+#### 1. list
+List and filter command requests.
+```sh
+$ espercli commandsV2 list [OPTIONS]
+```
+##### Options
+| Name, shorthand     | Default| Description|
+| ------------------- |:------:|:----------|
+| --commandtype, -ct  |        | Filter by type of command request |
```
Comment (tr0n-p46) [inline README.md:1949]: Use snake case

File: README.md [hunk 2]
```diff
@@ -1931,6 +1931,181 @@ parent       <uuid>
...
+| --json, -j          |        | Render result in JSON format |
+
+##### Example
+```sh
+$ espercli commandsV2 list -ct device -dt active -d DEV-ELOP-W57Z -l 2
+
+Total Number of Command Requests: 36
+
+RQUEST ID                             COMMAND               STATUS                                      ISSUED BY
+242f75a0-ea56-4cb6-898b-c017eaa01758  SET_KIOSK_APP       [{'state': 'Command TimeOut', 'total': 1}]  {'id': 12491, 'username': 'alok'}
```
Comment (tr0n-p46) [inline README.md:1964]: Can we avoid the json? Like just put Command TimeOut in status and alok in issued by?

Comment (BindyaBhaskar) [reply README.md:1964]: I had tried it for issued_by. But according to the spec, it is returned as a string and not json. So couldn't extract it. I can maybe do a spilt on the string and get the required response.

File: README.md (hunk 1 above)
Comment (BindyaBhaskar) [reply README.md:1949]: Okay

Comment (tarun91vas) [reply README.md:1949]: Also in description mention default values available for each parameter. And start with a simple example with minimum required options and then give an example by changing default variables.

File: README.md (hunk 2 above)
Comment (tarun91vas) [reply README.md:1964]: @BindyaBhaskar  you can parse string into json and then extract it.

File: README.md (hunk 1 above)
Comment (BindyaBhaskar) [reply README.md:1949]: Okay. Will do.

File: README.md [hunk 3]
```diff
@@ -1931,6 +1931,236 @@ parent       <uuid>
...
+
+
+##### Example
+```sh
+$ espercli commandsV2 command -c update_heartbeat -ct device -d DEV-ELOP-UULA
+
+TITLE          DETAILS
+Id             34dae2cb-a790-4912-b3a2-1630da8fea90
+Command        UPDATE_HEARTBEAT
+Command Args   {'app_state': None,
```
[the remark this answers is not in the document]
Comment (BindyaBhaskar) [reply README.md:2105]: The response is common for all the commands. One function was implemented for every command. We are not giving the command arguments in request.

File: README.md [hunk 4]
```diff
@@ -1931,6 +1931,236 @@ parent       <uuid>
...
+                'state': None,
+                'wifi_access_points': None}
+Command Type   DEVICE
+Devices        ['babc9cf5-2dbb-4382-bb9d-d6245941db35']
+Groups         []
+Device Type    active
+Status         []
+Issued by      {'id': 12000, 'username': 'bindya'}
+Schedule       IMMEDIATE
+Schedule Args  {'days': ['All days'],
```
[the remark this answers is not in the document]
Comment (BindyaBhaskar) [reply README.md:2121]: The schedule args are empty in the request. The response is common for all and hence is being displayed with None for parameters.

File: README.md [hunk 5]
```diff
@@ -1931,6 +1931,236 @@ parent       <uuid>
...
+                'message': None,
+                'package_name': None,
+                'policy_url': None,
+                'state': None,
+                'wifi_access_points': None}
+Command Type   DEVICE
+Devices        ['2d110b9c-6f65-430f-869d-fefb2a576dd3']
+Groups         []
+Device Type    all
+Status         [{'state': 'Command Scheduled', 'total': 1}]
```
[the remark this answers is not in the document]
Comment (BindyaBhaskar) [reply README.md:2148]: Did you mean to just display the state?

**Review by abdulMuneerM — APPROVED** [review]

**Review by tr0n-p46 — APPROVED** [review]

**Review by tarun91vas — APPROVED** [review]

**Pull request merged**